frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1037,9 +1077,11 @@ pub mod pallet {
 			let maybe_max_voters = maybe_max_voters.map(|x| x as usize);
 			let maybe_max_targets = maybe_max_targets.map(|x| x as usize);
 
+			// TODO: is it right to pass 0 remaining?
 			let supports = T::GovernanceFallback::elect_with_bounds(
 				maybe_max_voters.unwrap_or(Bounded::max_value()),
 				maybe_max_targets.unwrap_or(Bounded::max_value()),
+				0,
```

Comment:
should be `DataProvider::msp()`.

If we do a single page election, we want to make sure that we are using the "best" subset of voters, which is always `msp`.